src/components/App.tsx

```diff
@@ -35,24 +34,25 @@ function App() {
           </Link>
           .
         </Alert>
+        {/* What would we like to do with this feedback box ?*/}
```

Comment:
@kimberlythegeek @esanuandra If we still find it useful then perhaps we can incorporate the feedback message in the warning that we already display on the first page - `This is an unstable pre-release version. Some features may not yet be supported. Please file any bugs on the [Github Repo](https://github.com/mozilla/perfcompare/issues).` - I think it's better then having it hidden.

Reply:
Moved the leave feedback icon to top banner